doc/sphinx/credits.rst

```diff
@@ -1404,3 +1404,141 @@ The contacts of the Coq Consortium are Yves Bertot and Maxime Dénès.
 | Santiago de Chile, March 2018,
 | Matthieu Sozeau for the |Coq| development team
 |
+
+Credits: version 8.9
+--------------------
+
+|Coq| version 8.9 contains the result of refinements and stabilization
+of features and deprecations or removals of deprecated features,
+cleanups of the internals of the system and API along with a few new
+features. This release includes many user-visible changes, including
+deprecations that are documented in ``CHANGES.md`` and new features that
+are documented in the reference manual. Here are the most important
+changes:
+
+- Kernel: mutually defined records are now supported, by Pierre-Marie
+  Pédrot.
```

Comment:
Assuming I understand what this feature does and why it is here I think it may be worth adding `recursive` somewhere.